src/main/kotlin/no/nav/k9/tjenester/kokriterier/KøkriterierDto.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.k9.tjenester.kokriterier
+
+data class KøkriterierDto(
```

Comment:
Er litt usikker på om denne strukturen.. Hva tenker du @qtips ?
Om du har et forslag til et bedre navn, tas det gjerne i mot 😄

Reply:
Ser du endte opp med å fjerne den. Jeg tenker at det kan være fint å ha kriteriene som en egen type med eget endepunkt en gang i fremtiden, for å kunne legge til flere attributter som f.eks type seleksjon som skal brukes i frontend, men kan vente til vi vet mer om hvordan vi vil ha det.

Reply:
Ja, det var det Qadeer og jeg snakket om i går. Vi kan kanskje prate litt mer om det i dag